lib/ae_mdw/db/rocksdb_cf.ex

```diff
@@ -240,7 +240,7 @@ defmodule AeMdw.Db.RocksDbCF do
           end
 
         :not_found ->
-          last_key(table)
+          :not_found
```

Comment:
Why does `iterator_prev_key` defaults to `:not_found`, but `iterator_next_key` defaults to `first_key(tab)`? I don't think we should have any asymmetric behavior in that sense

Reply:
an error that we haven't before